source/crud/tests/v2/bulkWrite-arrayFilters.yml

```diff
@@ -65,3 +65,60 @@ tests:
         data:
           - {_id: 1, y: [{b: 2}, {b: 2}]}
           - {_id: 2, y: [{b: 0}, {b: 2}]}
+  -
+    description: "BulkWrite with arrayFilters using update pipelines"
+    operations:
+      -
+        name: "bulkWrite"
+        arguments:
+          requests:
+            -
+              # UpdateOne when one document matches arrayFilters
+              name: "updateOne"
+              arguments:
+                filter: {}
+                update: [ { $set: { "y.$[i].b": 2 } } ]
+                arrayFilters: [ { "i.b": 3 } ]
```

Comment:
I'm not familiar with the feature, but does the set of `arrayFilters` simply apply to _all_ updates in the pipeline now (where it previously only applied to one)?

Reply:
`arrayFilters` is not a supported option with pipelines. I have removed them from the update pipelines tests.